packages/expo-video/src/VideoView.web.tsx

```diff
@@ -60,17 +93,32 @@ class VideoPlayerWeb implements VideoPlayer {
     this.isPlaying = true;
   }
   seekBy(seconds: number): void {
-    this.mountedVideos.forEach((video) => {
+    this._mountedVideos.forEach((video) => {
       video.currentTime += seconds;
     });
   }
   replay(): void {
-    this.mountedVideos.forEach((video) => {
+    this._mountedVideos.forEach((video) => {
       video.currentTime = 0;
       video.play();
     });
     this.isPlaying = true;
   }
+
+  _addListeners(video: HTMLVideoElement): void {
+    video.onloadedmetadata = () => {
+      const source = audioContext.createMediaElementSource(video);
+      this._audioNodes.add(source);
+
+      // First mounted video should be connected to the audio context. All other videos have to be muted.
+      if (this._audioNodes.size === 1) {
+        source.connect(audioContext.destination);
+      } else {
+        source.connect(zeroGainNode);
+      }
+      source.connect(zeroGainNode);
```

Comment:
Do we need to call `source.connect` twice?

Reply:
This looks like a leftover, thanks 😄